### PR TITLE
  Add X509_get0_authority_key_id() function

### PR DESCRIPTION
**Pull request by Romain-Geissler-1A**
Hi,

This is a rebase of #4595 (with just the part about authority_key_id).

Cheers,
Romain

### CODE REVIEW

File: util/libcrypto.num [hunk 1]
```diff
@@ -4501,3 +4501,4 @@ EVP_sha512_256                          4442	1_1_1	EXIST::FUNCTION:
 EVP_sha512_224                          4443	1_1_1	EXIST::FUNCTION:
 OCSP_basic_sign_ctx                     4444	1_1_1	EXIST::FUNCTION:OCSP
 RAND_DRBG_bytes                         4445	1_1_1	EXIST::FUNCTION:
+X509_get0_authority_key_id              4446	1_1_1	EXIST::FUNCTION:
```
Comment (kroeckx) [inline util/libcrypto.num]: Since this is going to be backported to 1.1.0, this is wrong. It think it should say 1_1_0h. Not sure about the numbers.

Comment (richsalz) [reply util/libcrypto.num]: dang, you are right.

Comment (Romain-Geissler-1A) [reply util/libcrypto.num]: I have rebased this pull request on top of current master, and switched the version number of X509_get0_authority_key_id  to 1_1_0h.

**Comment by Romain-Geissler-1A** [comment]
Looks like Appveyor tests are red, but I don't really know why (only one out of the 4 builds is broken).

**Comment by richsalz** [comment]
The failure is unrelated; https://ci.appveyor.com/project/openssl/openssl/build/master.15720/job/ab8iktg2aur952th#L35252 

The assigned number is free in 1.1.0, so this can cherry-pick back easily.  Still approved.  Ping committers.

**Comment by Romain-Geissler-1A** [comment]
Sorry for my misunderstanding, I don't know how you usually work in openssl. I have to ping people with push rights on OpenSSL to get this merged/cherry-pick ? Who are these people (I mean except you ?). Do I still need a second review now that two people reviewed it already ?

**Comment by richsalz** [comment]
No problem.  Our workflow is that two members of the openssl-committers have to approve the PR, and one of them must be in the project management (OMC) group.  Then we will push the change.  This is basically how Apache projects work, too.

Thank you for your contribution, and improving openssl!

**Comment by richsalz** [comment]
Pushed to master and 1.1.0; thanks!

**Pull request closed without merging**